src/Nest/CommonAbstractions/Union/UnionJsonConverter.cs

```diff
@@ -12,8 +12,6 @@ internal class UnionJsonConverter : JsonConverter
 		private static readonly ConcurrentDictionary<Type, UnionJsonConverterBase> KnownTypes = new ConcurrentDictionary<Type, UnionJsonConverterBase>();
 
 		public override bool CanConvert(Type objectType) => true;
-		public override bool CanRead => true;

```

Comment:
while not needed I like the explicit self documenting implementations of implementing `CanRead` and `CanWrite`